egs/fisher_callhome_spanish/st1/local/normalize_trans.sh

```diff
@@ -14,7 +14,7 @@ if [ $# -lt 2 ]; then
 fi
 
 # download data preparation scripts for transcriptions
-[ ! -d data/local/fisher-callhome-corpus ] && git clone https://github.com/joshua-decoder/fisher-callhome-corpus.git data/local/fisher-callhome-corpus
+[ ! -d data/local/fisher-callhome-corpus ] && git clone https://github.com/siddalmia/fisher-callhome-corpus.git data/local/fisher-callhome-corpus
```

Comment:
Did you modify it from your fork?
If so, it is better to leave some comments here.

Reply:
Added. Thanks!